### ISSUE TITLE
More visibility to the example reading a CSV file in small chunks

### ISSUE THREAD

**Issue by ppcano** [grafana/k6-docs#1650]
Reading large files can lead to OOM errors, which is a common problem with `open()`  (here a minor [related PR](https://github.com/grafana/k6-docs/pull/1649)) . 

The [example using the new Stream API](https://grafana.com/docs/k6/latest/javascript-api/k6-experimental/streams/#example)  provides a solution for this.

Wondering how we could better connect this example with other parts of the docs or additional tutorials.

**Comment by joanlopez** [comment]
Yeah! I always thought that we have very good material within the "Examples" section, but unfortunately its visibility is not very good (imho). So, I guess we could explore things like:

- (Now, and from time to time), do a pass over all the examples and find all the places they could be linked to. For instance, the "HTTP Authentication" example could be linked from the [HTTP module docs](https://grafana.com/docs/k6/latest/javascript-api/k6-http/).
- Try to find another, better place for such kind of useful resources. Like we have "Recommended practices" specifically for the Browser section. Perhaps a "Troubleshooting" section, where one of the topics is memory management?

Beyond that, it would also be very good to have better discoverability through the Search functionality, but afaik that's under discussion right now.
